setup.py

```diff
@@ -435,7 +435,7 @@ def run(self):
             "MACOSX_DEPLOYMENT_TARGET", current_system
         )
         if (
-            LooseVersion(python_target) < "10.9"
+            LooseVersion(str(python_target)) < "10.9"
```

Comment:
check if https://docs.python.org/3/library/sysconfig.html#module-sysconfig returns a bytestring (e.g. `get_config_vars()`)

weird

Reply:
int 11

Reply:
hmm that's also weird